cmake/external/neural_speed.cmake

```diff
@@ -0,0 +1,20 @@
+set(BTLA_URL https://github.com/intel/neural-speed.git)
```

Comment:
The repo needs to have a license file.

Reply:
updated in 009adb61a3c3d7d52eb1ec4c1afa5fa43df51859